typing/jkind.ml

```diff
@@ -505,6 +505,47 @@ module Mod_bounds = struct
     && Nullability.equal (nullability t1) (nullability t2)
     && Separability.equal (separability t1) (separability t2)
 
+  (* [diff base actual] returns the axes on which [actual] is strictly
```

Comment:
Should also say that this returns `None` if `actual` is weaker than `base`.

Nit pick: use `(** ... *)` comment so this annotates the function.